XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Pointers/LinePointer.cs

```diff
@@ -176,24 +192,84 @@ public override void OnPostRaycast()
 
             // Used to ensure the line doesn't extend beyond the cursor
             float cursorOffsetWorldLength = BaseCursor?.SurfaceCursorDistance ?? 0f;
+            
+            if (IsSelectPressed)
+            {
+                BeamAnimator.SetBool("triggerPull", true);
```

Comment:
these animator strings can be updated to use the string to id for better performance, but also for customization for other developers using the module